context-free.md

```diff
@@ -2,21 +2,31 @@
 
 ##  Who is this programming language for?
 
+This programming language is for artists. It allows artists to create complex pattern art with thousands of repeated shapes that might otherwise be tedious and time consuming to create.
+
 
 ## What is easy to do in this language? Why is it easy?
 
+It is easy to create patterns. Once you have defined your own shape (or using a predefined shape), Context Free only requires one or two commands to have the shape copied across the screen in different colors, sizes, brightness, at different angles, or with any number of other small variations.
+
 
 ## What is hard to do in this language? Why is it hard?
 
+Drawing something like a person, or another irregular shape/scene. Context Free works very well for branching, repetitive shapes in abstract art, but more realistic art requires many unique, irregular shapes and it would be difficult and time consuming for an artist to program in each unique shape.
 
 ## How did you learn how to program in this language?
 
+I looked up various example code and artwork, modified it, and observed the changes that occurred until I understood the effect of each parameter. I then created my own.
+
 
 ## What is the underlying _computational model_ for this programming language? 
 _We don't yet have a great definition of the term "computational model". 
 For now, try to come up with the clearest, most concise explanation of what 
 happens when a ContextFree program runs._
 
+A program begins with the starting shape and expands each of the rules it encounters independently until all that is left are the terminal, built in shapes which are then drawn.
+
```

Comment:
I agree, this is basically what I said. I'm curious how the language deals with the different modifiers to each shape (_i.e._ where position, hue, etc. come into play).